synapseclient/models/dataset.py

```diff
@@ -1095,67 +963,54 @@ async def add_item_async(
 
         Arguments:
             item: Entity to add to the dataset. Must be an EntityRef, File, or Folder.
-            synapse_client: If not passed in and caching was not disabled by
-                `Synapse.allow_client_caching(False)` this will use the last created
-                instance from the Synapse class constructor.
+            synapse_client: The Synapse client to use. Defaults to None.
```

Comment:
```suggestion
            synapse_client: If not passed in and caching was not disabled by
                `Synapse.allow_client_caching(False)` this will use the last created
                instance from the Synapse class constructor.
```